README.md

```diff
@@ -8,4 +8,23 @@ To build this, you need the following:
 - [`latexmk`](https://ctan.org/pkg/latexmk?lang=en) which may come with your LaTeX distribution
 - [Graphviz](https://graphviz.org/)
 
-After resolving these dependencies, run `make` to compile `spec.md` into `build/spec.pdf`.
+For compatability with continuous integration (CI) testing, use the versions of
```

Comment:
```suggestion
For compatibility with continuous integration (CI) testing, use the versions of
```

Reply:
Fixed in: https://github.com/chipsalliance/firrtl-spec/pull/320/commits/1c37c37efdfd45dfaa1764044627e25284abe43d